src/test/e2e/api/admin/project/features.e2e.test.ts

```diff
@@ -2703,7 +2703,7 @@ test.only('should add multiple segments to a strategy', async () => {
                 (strat) => strat.id === strategyOne.id,
             );
 
-            expect(strategy.segments).toEqual([
+            expect(strategy.segments.sort()).toEqual([
```

Comment:
We're sorting in the test, so we're basically saying: the order is not important. If it is, we should consider a different change